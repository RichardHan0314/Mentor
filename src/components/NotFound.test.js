import React from 'react';
import { render, screen } from '@testing-library/react';
import NotFound from './NotFound';

describe('NotFound component', () => {
    it('Render NotFound snapshot', () => {
        const { asFragment } = render(<NotFound />);
        expect(asFragment(<NotFound />)).toMatchSnapshot();
    });

    it('Render NotFound correctly', () => {
        render(<NotFound />);
        // screen.debug();
        expect(screen.getByText('No Movie Found')).toBeInTheDocument();
    });
});