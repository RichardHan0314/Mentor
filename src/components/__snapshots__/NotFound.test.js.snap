// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component Render NotFound snapshot 1`] = `
<DocumentFragment>
  <p
    class="sc-AxjAm dVuOJJ"
  >
    No Movie Found
  </p>
</DocumentFragment>
`;
